app/models/question.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: questions
+#
+#  id           :bigint           not null, primary key
+#  for_banks    :boolean
+#  for_partners :boolean
+#  title        :string
```

Comment:
To ensure higher data integrity, please add database constraints so that even though the application fails to validate, the database will still enforce it.

https://thoughtbot.com/blog/validation-database-constraint-or-both is a good resource for learning about why and how to add them.

Reply:
I added some database constraints. Please let me know if it still needs adjusting.